aspnetcore/performance/caching/memory.md

```diff
@@ -21,8 +21,12 @@ ASP.NET Core supports several different caches. The simplest cache is based on t
 
 Non-sticky sessions in a web farm require a [distributed cache](distributed.md) to avoid cache consistency problems. For some apps, a distributed cache can support higher scale-out than an in-memory cache. Using a distributed cache offloads the cache memory to an external process.
 
+::: moniker range="<= aspnetcore-2.0"
```

Comment:
We made this change in 2.0 so I think this should be strictly less than instead of less than or equal to.